doc/manual.asciidoc

```diff
@@ -533,7 +533,7 @@ To create a rule that never rebuilds, use a build rule without any input:
 rule touch
   command = touch $out
 build file_that_always_exists.dummy: touch
-build dummy_target_to_follow_a_pattern: phony file_that_always_exists.dummy
+build dummy_target: phony file_that_always_exists.dummy
```

Comment:
???

Reply:
This line is overflowing its code block.

Reply:
I see. Still I think this should be handled with a scrollbar instead.